CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ All notable changes to this project will be documented in this file.
 
 ### :bug: (Bug Fix)
 
+* fix(sdk-trace-base): make span start times resistant to hrtime clock drift
```

Comment:
Changelog entry was created before the release and needs to be in the unreleased section. :slightly_smiling_face: